docs/en/quick-answers/troubleshooting/turn-debug-logging-on.md

```diff
@@ -95,6 +93,8 @@ engines:
 
 Save the file and relaunch Nuke in the shot step environment. You will now see the debug output in the script editor window. 
 
+{% include info title="Note" content="If debug logging is enabled via any one of the Desktop checkbox, environment variable, or engine configuration setting, debug logging will be output. Additionally, each of these three is modified independently of the others: the checkbox value is a QSetting, and it persists across sessions of Desktop via QSetting, totally independent of the engine setting or env var. This means that while the Desktop checkbox may be unchecked, debug logging might still be enabled by way of one of the other methods." %}
```

Comment:
```suggestion
{% include info title="Note" content="If debug logging is enabled via any one of the Desktop checkbox, environment variable, or engine configuration setting, debug logging will be output. Additionally, each of these three is modified independently of the others: the checkbox state is stored persistently across sessions via the use of `QSettings`, totally independent of the engine setting or environment variable. This means that while the Desktop checkbox may be unchecked, debug logging might still be enabled by way of one of the other methods." %}
```

Reply:
`QSetting` isn't a thing, but `QSettings` is, and it doesn't technically return a `QSetting` value, I think it returns a `dict` with `PySide` and a `QVariant` with `PyQt` (but we don't use `PyQt` with Desktop so kinda irrelevant 😛)

Do we need to mention again that the Desktop checkbox is stored persistently, we already say that at the top? If you think important to remind them, then perhaps keep it simpler and not bother mentions the details of `QSettings` at all, and maybe bump that bit to the top if you think that's still needed.
Other than that, I think this note box is good info!

Reply:
omitted the QSettings details, cleaned up the wording, Desktop -> Shotgun Desktop.